test/extended/apiserver/graceful_termination.go

```diff
@@ -253,6 +255,8 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer][Late]", func() {
 			lateRequestCounter := 0
 			readFile := false
 
+			klog.Infof("Starting to scan logs for API server: %s", apiServerName)
```

Comment:
FYI - There is a [monitor test flavor of this test](https://github.com/openshift/origin/blob/95b5488acfc75590d69b0bc94354ef6cb6c5dcc6/pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go#L345) as well